packages/components/src/menu/src/components/osds-menu/core/controller.ts

```diff
@@ -86,10 +86,6 @@ class OdsMenuController {
     case 'ArrowDown':
     case 'Tab':
       return this.handlerKeyArrow(event, selectedMenuItemIndex);
-    case 'Enter':
```

Comment:
I don't think the menu trigger is an `osds-button` so this will just remove the whole key trigger

Reply:
Fixed